### ISSUE TITLE
Bug with `clone item bracket`

### ISSUE THREAD

**Issue by SimeonC** [cursorless-dev/cursorless#944]
Using the following code, select the last `calculateValue` key in the `classUpdateDefinitions` then say `clone item bracket`.

* **Expected** The element in the `classUpdateDefinitions` array is cloned.
* **Actual** The entire content of `Mixin.create` is cloned.

![CleanShot 2022-09-09 at 17 56 54](https://user-images.githubusercontent.com/1085899/189313082-1e9f116e-0672-486d-900a-0ff2db5204ab.gif)

```js
import Ember from 'ember';

const classUpdateDefinitions = [
  {
    className: 'table-future-extended',
    mapKeys: [{ mapKey: 'ongoingReservationMap', watchKeys: ['isFutureExtended'] }],
    calculateValue: (self, tableId) => self.get(`ongoingReservationMap.${tableId}.isFutureExtended`)
  },
  {
    className: 'table-past-extended',
    mapKeys: [{ mapKey: 'ongoingReservationMap', watchKeys: ['isPastExtended'] }],
    calculateValue: (self, tableId) => self.get(`ongoingReservationMap.${tableId}.isPastExtended`)
  },
  {
    className: 'table-locked',
    mapKeys: [{ mapKey: 'ongoingReservationMap', watchKeys: ['hasLockedTables'] }],
    calculateValue: (self, tableId) => self.get(`ongoingReservationMap.${tableId}.hasLockedTables`)
  },
  {
    className: 'disable-table-update',
    mapKeys: [{
      valueKey: 'currentTimetableable',
      watchKeys: [
        'hasLockedTables',
        'isGeneric',
        'isReservation',
        'isWalkin',
        'isNew',
        'tables.[]',
        'origTables.[]'
      ]
    }],
    calculateValue: (self, tableId) => self.shouldDisableTableUpdate(tableId)
  },
  {
    className: 'table-selected-original',
    mapKeys: [{
      valueKey: 'currentTimetableable',
      watchKeys: [
        'origTables.[]'
      ]
    }],
    calculateValue: (self, tableId) => self.isSelectedOriginal(tableId)
  }
];

const FloorSectionTableClasses = Ember.Mixin.create({
  init() {
    this._super(...arguments);
    classUpdateDefinitions.forEach(({ className, calculateValue }) => {
      this[`updateClass-${className}`] = (_, observerName) => {
        const id = observerName.split('.')[1];
        this.toggleTableClass(id, className, calculateValue(this, id));
      };
    });
  },

  onSelectedTimetableableTablesChange: Ember.observer('selectedTimetableable.tables.[]', () => {
    const selectedTables = this.get('selectedTimetableable.tables') || [];
    this.get('sectionTables').forEach((table) => {
      this.toggleTableClass(table.get('id'), 'table-locked', selectedTables.includes(table));
    });
  }),

  toggleTableClass(tableId, className, isActive) {
    this.$().find(`[data-table-class-target="${tableId}"]`).toggleClass(className, isActive);
  },

  registerClassObservers(tableId) {
    classUpdateDefinitions.forEach(({ className, mapKeys }) => {
      mapKeys.forEach(({ mapKey, valueKey, watchKeys }) => {
        watchKeys.forEach((watchKey) => {
          const observerKey = valueKey ? `${valueKey}.${watchKey}` : `${mapKey}.${tableId}.${watchKey}`;
          Ember.addObserver(this, observerKey, this[`updateClass-${className}`]);
        });
      });
    });
  }
});

export default FloorSectionTableClasses;
```

**Comment by pokey** [comment]
Sorry this gif is really hard to follow.  Mind adding more detail to the description, or re-recording the gif?  I've watched it about 10 times and can't make heads or tails of it 😅

**Comment by SimeonC** [comment]
Oh, my bad. Looks like I completely cut the end off it. On Monday I'll try raise a PR with a failing test - that'll probably be far more useful than a recording.

**Comment by pokey** [comment]
Sure. It's probably worth searching the issues for "item" first, though, to make sure it's not a duplicate, if you haven't already

**Comment by SimeonC** [comment]
Raised a PR with a failing test, that should help out more than this issue I think.

**Comment by SimeonC** [comment]
My fault here, got confused between pairs and bracket so PBKAC not a bug
